mtags/src/main/scala-2/scala/meta/internal/pc/WorkspaceSymbolSearch.scala

```diff
@@ -183,13 +183,11 @@ trait WorkspaceSymbolSearch { compiler: MetalsGlobal =>
   }
 
   private def loadSymbolFromClassfile(
-      classfile: SymbolSearchCandidate
+      classfile: SymbolSearchCandidate,
+      context: Context
   ): List[Symbol] = {
     def isAccessible(sym: Symbol): Boolean = {
-      sym != NoSymbol && {
-        sym.info // needed to fill complete symbol
-        sym.isPublic
-      }
+      context.isAccessible(sym, sym.info)
```

Comment:
Do we need it here if it's already in AutoImportsProvider? Could we instead just leave `sym != NoSymbol`?

I am mostly thinking that sym.info might be a bit costly (not 100% sure how much), so we can verify it once this becomes a candidate at the end.

Reply:
That's a good question. This implementation looks to be used as part of completion ([one usage](https://github.com/scalameta/metals/blob/d91f489f5595fbb981080d9fd8383ade1ba69ce2/mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala#L431) so I thought we should still have some visibility filtering here.

Reply:
_but_ maybe it _should_ be the responsibility of the caller to "filter" (e.g. as part of the passed in `visit` function)?

Reply:
Either way, I might trying to optimise too early here. I would leave `isAccessible` just here or move it downstream to the actual callers of the visitor

Reply:
Alright. For now I will just leave things as they are.